.yamllint

```diff
@@ -6,17 +6,25 @@ ignore: |
   .github/
   # Generated file
   tests/files/custom_cni/cilium.yaml
-
+# https://ansible.readthedocs.io/projects/lint/rules/yaml/
 rules:
   braces:
     min-spaces-inside: 0
     max-spaces-inside: 1
   brackets:
     min-spaces-inside: 0
     max-spaces-inside: 1
+  comments:
+    # https://github.com/prettier/prettier/issues/6780
+    min-spaces-from-content: 1
+  # https://github.com/adrienverge/yamllint/issues/384
+  comments-indentation: false
```

Comment:
Indented comments seems to be a nice thing overall even with the behavior pointed in the issue so not sure about disabling it tbh